src/components/JobsPage/Jobs.js

```diff
@@ -142,8 +150,9 @@ const Jobs = ({
           },
           spec: {
             function: job.function,
-            handler:
-              Object.values(functionData?.spec.entry_points)?.[0]?.name ?? '',
+            handler: functionData?.spec?.entry_points
+              ? Object.values(functionData?.spec.entry_points)?.[0]?.name
+              : '',
```

Comment:
I suggest doing this:
```suggestion
            handler:
              Object.values(functionData?.spec?.entry_points ?? {})[0]?.name ??
              '',
```

If `functionData` or `functionData.spec` or `functionData.spec.entry_points` are nullish, the `?? {}` would give `{}`. Then `Object.values({})` gives `[]`. Then no need for `?.[0]`, we can simply use `[0]` because there will always be an array there. If the array is empty, `[0]` will give `undefined`. `undefined?.name` still returns `undefined`. Then `undefined ?? ''` will give `''`.
Bottom line, if `functionData.spec.entry_points` is a non-empty object, its first item's name is returned. Otherwise an empty string is returned.